src/types.ts

```diff
@@ -192,6 +198,33 @@ export type NoteSubmit = {
   ParentId: string;
 };
 
+export type ActivitySubmit = {
```

Comment:
This should come from the metadata of the sObject - so I don't think you can have a concrete type